Apps/TicketManagement/Services/RedisTicketService.cs

```diff
@@ -171,43 +174,17 @@ private static string GetRoomName(RoomConfiguration config, string roomType)
             return $"{{{config.Name}}}:Room:{roomType}";
         }
 
-        private (string Token, long Expires) CreateJwt(RoomConfiguration roomConfig, Guid id)
+        private async Task<(string Token, long Expires)> CreateJwt(RoomConfiguration roomConfig)
```

Comment:
"Async" should be appended to method name.

Reply:
I've updated all of the methods to include async